src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+.stars {
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+
+  &__star--active {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star-active.svg);
+    box-sizing: border-box;
+    align-items: center;
+    margin: 0 4px 0 0;
+    background-position: center;
+    background-repeat: no-repeat;
```

Comment:
Consider creating a mixin for the common styles between `.stars__star--active` and `.stars__star`. Since they share multiple properties (width, height, box-sizing, align-items, margin, background-position, background-repeat), a mixin will help avoid duplication and make the code easier to maintain.